spec/src/main/asciidoc/Preface.adoc

```diff
@@ -1,32 +1,32 @@
 == Preface
 
-This is the Jakarta Faces Specification Document, version 4.0.
+This is the Jakarta Faces Specification Document, version 4.1.
 
 
 === Related Technologies
 
 [[a173]]
 ==== Other Jakarta Platform Specifications
 
-Jakarta Faces 4.0 is based on the following Jakarta specifications:
+Jakarta Faces 4.1 is based on the following Jakarta specifications:
 
 
-Jakarta Servlet, version 6.0
+Jakarta Servlet, version 6.1
 <https://jakarta.ee/specifications/servlet/>
 
-Jakarta Expression Language, version 5.0
+Jakarta Expression Language, version 6.0
 <https://jakarta.ee/specifications/expression-language/>
 
-Jakarta Contexts and Dependency Injection, version 4.0
+Jakarta Contexts and Dependency Injection, version 4.1
 <https://jakarta.ee/specifications/cdi/>
 
-Jakarta Bean Validation, version 3.0
+Jakarta Bean Validation, version 3.1
 <https://jakarta.ee/specifications/bean-validation/>
 
-Jakarta Tags, version 2.0
+Jakarta Tags, version 3.0
```

Comment:
Are we sure we are supporting this?

Reply:
Hmm interesting, this is indeed not explicitly set in pom.xml. Faces does however provide the `<c:xxx>` and `${fn:xxx}` implementations for Facelets. So the Jakarta Tags API is implied.

Reply:
@BalusC I think it piggy backed on it and now that Pages is no longer supported this specification should not be either. 

What does that mean for those tags? They should be in the Faces specification, just like the h: and f:

Reply:
You are right, good catch. Faces simply emulates them, but doesn't depend on Jakarta Tags, nor does it use any of its types.

Reply:
I just recalled that I once experienced that Tomcat failed to deploy a JSF+Facelets app when JSTL API dependency was left out from pom.xml. I'll have to take a second look if that's still reproducible on Faces 4.1 and if so, why exactly.

Reply:
@BalusC We really should not refer to it as this really was about JSP support. The Facelets variants work can now be free to progress at their own pace and they should be completely specified and owned by the Faces specification.